machine/machine.py

```diff
@@ -146,7 +146,8 @@ def ls(self):
         cmd = ["ls", "-f", fields]
         stdout, stderr, errorcode = self._run(cmd)
         machines = []
-        for line in stdout.split("\n"):
+        lines = (l for l in stdout.split("\n") if len(l) > 0)
```

Comment:
maybe better to do `stdout.strip.split("\n")` ?

Reply:
Nice suggestion! 
Done in https://github.com/gijzelaerr/python-docker-machine/pull/15

Feel free to discard this PR